target/linux/ramips/dts/mt7621_cudy_wr2100.dts

```diff
@@ -0,0 +1,229 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "cudy,wr2100", "mediatek,mt7621-soc";
+	model = "Cudy WR2100";
+
+	aliases {
+		led-boot = &led_blue;
+		led-failsafe = &led_blue;
+		led-running = &led_blue;
+		led-upgrade = &led_blue;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_blue: blue {
+			label = "blue:internet";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+		};
+
+		red {
+			label = "red:internet";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+
+		wan {
+			label = "green:wan";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+
+		lan1 {
+			label = "green:lan1";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		lan2 {
+			label = "green:lan2";
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+		};
+
+		lan3 {
+			label = "green:lan3";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+
+		lan4 {
+			label = "green:lan4";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio 10 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+};
+
+&gmac0 {
+	mtd-mac-address = <&bdinfo 0xde00>;
+	phy-mode = "trgmii";
+};
+
+// &gmac1 {
```

Comment:
stock uses both gmacs to connect to the switch:
```
VLAN 1:
        vid: 1
        ports: 0 1 2 3 6 
VLAN 2:
        vid: 2
        ports: 4 5
```
this is unfit for openwrt, right?
i've kept the definitions for `gmac1` and `port@5` in comments. should i delete them?

Reply:
Either you need them or you remove them.

Reply:
i'm removing it, then.

for future reference and too allow people to make use of the additional link more easily, here are the removed nodes:

```
&gmac1 {
	status = "okay";

	mtd-mac-address = <&bdinfo 0xde00>;
	phy-mode = "rgmii"; // TODO overrides rgmii-rxid

	fixed-link {
		speed = <1000>;
		full-duplex;
		pause;
	};
};

&switch0 {
	ports {
		port@5 {
		 reg = <5>;
		 label = "cpu2";

		 ethernet = <&gmac1>;
		 phy-mode = "rgmii";

		 fixed-link {
			 speed = <1000>;
			 full-duplex;
		 };
		};
	}
}
```